### PR TITLE
feat: resolve studio urls in frontend

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** There are a fair number of "unresolved" studio URLs present in production ORA prompt definitions. Rather than attempting to fix them, we've stopped them from being introduced in other fixes, and with this are attempting to resolve them as they are displayed.

JIRA: [AU-555](https://openedx.atlassian.net/browse/AU-555)

**What changed?**

- Added "base_asset_url" to response_mixin context which is inserted into the DOM as a data attribute in the response template
-  Created a Prompts class and Prompt js class to interface with all prompts / individual prompts respectively
- Added logic to ResponseView to, on load, attempt to resolve and re-display the prompt content with resolved links.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

- Download this course ([course.tar.gz](https://github.com/openedx/edx-ora2/files/8352064/course.tar.gz)), and import it into your local Studio on master. It contains one ORA.
- Navigate to the ORA in the LMS and Studio and observe the different behaviors of links / images / media.
- Check out this branch, install-local, and then attach and restart both lms and studio.
- Navigate to the ORA in the LMS and Studio and observe the new behavior and how it affects links / images / media.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by jansenk** [comment]
Updated the course zip in the description

**Review by mattcarter — APPROVED** [review]
Approved, but still needs the version bump

File: openassessment/xblock/static/js/src/lms/oa_prompts.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+    this.staticUrl = '/static/';
+    this.baseAssetURL = sel.data('baseAssetUrl');
+  }
+
+  /**
+  * For each prompt, attempt to replace static studio URLs that may
+  * have made their way into the prompt with their resolved web versions.
+  */
+  resolveStaticLinks() {
```
Comment (muselesscreator) [inline openassessment/xblock/static/js/src/lms/oa_prompts.js:58]: prefer for this to be either a bound method or an arrow function to ensure the preservation of the local `this` context.

**Review by muselesscreator — APPROVED** [review]
Mild forward-looking concern about `this` context fragility, but largely LGTM
